packages/react-testing/src/tests/element.test.tsx

```diff
@@ -588,7 +588,7 @@ describe('Element', () => {
         defaultRoot,
       );
 
-      expect(() => element.triggerKeypath('actions')).toThrow(
+      expect(() => element.triggerKeypath('actions' as never)).toThrow(
```

Comment:
I think it'd be better to use a `@ts-expect-error` directive to describe why this line is not considered valid typescript rather than casting it to a nonsense value.

```suggestion
        // @ts-expect-error actions is not valid parameter, because it does not point to a function
      expect(() => element.triggerKeypath('actions')).toThrow(
```

---

In the test block above this we have:

```ts
expect(() => element.triggerKeypath('actions[1]onAction')).toThrow(
  /Attempted to access field keypath 'actions\.1', but it was not an object/,
);
```

I would expect that would also a require a similar treatment as it also throws an error, thanks to passing an invalid string parameter . `actions[1]onAction` and `actions` are both invalid strings to pass into `triggerKeypath` - I would expect both of them to raise a type error instead of only `actions`.